server/app.js

```diff
@@ -176,13 +175,20 @@ const createApp = ({
     ),
   );
 
-  // GovUK Template Configuration
-  const establishmentId = getEstablishmentId(config.establishmentName);
-  app.locals.asset_path = '/public/';
-  app.locals.config = {
-    ...config,
-    establishmentId,
-  };
+  app.use((req, res, next) => {
+    const replaceUrl = /-prisoner-content-hub.*$/g;
```

Comment:
I'm loving where this is going 🙌. Would it be worth us adding a line to the readme that calls out that we do hostname splitting for staging/dev environments so the expectation for deploys is obvious?

Reply:
Makes sense 😄

Reply:
Done 😄